docs/sources/analyzing-profiles-in-pyroscope-ui/_index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "How to use the Pyroscope UI for performance analysis"
+menuTitle: "How to use the Pyroscope UI"
+description: "How to use the Pyroscope UI to analyze performance of your applications."
+weight: 30
+keywords:
+  - pyroscope
+  - UI
+  - performance analysis
+  - flamegraphs
+---
+
+# How to use the Pyroscope UI for performance analysis
+
+## Introduction
+
+**Understanding Continuous Profiling and Metadata in Pyroscope**
```

Comment:
This is actually a heading. Is continuous profiling and metadata unique to Pyroscope? Or is this going from a general discussion of continuous profiling and metadata to a narrower usage with Pyroscope? 
```suggestion
##  Continuous profiling and metadata 
```